modules/exploits/multi/http/nostromo_code_exec.rb

```diff
@@ -0,0 +1,86 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = GoodRanking
+
+  include Msf::Exploit::CmdStager
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Nostromo Remote Command Execution',
```

Comment:
Generic name. Perhaps:

```suggestion
      'Name'           => 'Nostromo Directory Traversal Remote Command Execution',
```